src/views/datasources/dataimportcron/hooks/useDataImportCronActions.tsx

```diff
@@ -0,0 +1,115 @@
+import { Action, k8sDelete, k8sPatch } from '@openshift-console/dynamic-plugin-sdk';
+
+type UseDataImportCronActionsProvider = (DataImportCron: V1beta1DataImportCron) => Action[];
+
+export const useDataImportCronActionsProvider: UseDataImportCronActionsProvider = (
+  dataImportCron,
+) => {
+  const { t } = useKubevirtTranslation();
+  const { createModal } = useModal();
+  const history = useHistory();
+
+  return React.useMemo(
+    () => [
+      {
+        id: 'dataimportcron-action-edit-labels',
+        disabled: false,
+        label: t('Edit labels'),
+        cta: () =>
+          createModal(({ isOpen, onClose }) => (
+            <LabelsModal
+              obj={dataImportCron}
+              isOpen={isOpen}
+              onClose={onClose}
+              onLabelsSubmit={(labels) =>
+                k8sPatch({
+                  model: DataImportCronModel,
+                  resource: dataImportCron,
+                  data: [
+                    {
+                      op: 'replace',
+                      path: '/metadata/labels',
+                      value: labels,
+                    },
+                  ],
+                })
+              }
+            />
+          )),
+      },
+      {
+        id: 'dataimportcron-action-edit-annotations',
+        disabled: false,
+        label: t('Edit annotations'),
+        cta: () =>
+          createModal(({ isOpen, onClose }) => (
+            <AnnotationsModal
+              obj={dataImportCron}
+              isOpen={isOpen}
+              onClose={onClose}
+              onSubmit={(updatedAnnotations) =>
+                k8sPatch({
+                  model: DataImportCronModel,
+                  resource: dataImportCron,
+                  data: [
+                    {
+                      op: 'replace',
+                      path: '/metadata/annotations',
+                      value: updatedAnnotations,
+                    },
+                  ],
+                })
+              }
+            />
+          )),
+      },
+      {
+        id: 'dataimportcron-action-edit-DataImportCron',
+        disabled: false,
+        label: t('Edit DataImportCron'),
+        cta: () =>
+          history.push(
+            `/k8s/ns/${dataImportCron.metadata.namespace || 'default'}/${DataImportCronModelRef}/${
+              dataImportCron.metadata.name
+            }/yaml`,
+          ),
+      },
+      {
+        id: 'dataimportcron-action-delete',
+        label: t('Delete DataImportCron'),
+        cta: () =>
+          createModal(({ isOpen, onClose }) => (
+            <DeleteModal
+              obj={dataImportCron}
+              isOpen={isOpen}
+              onClose={onClose}
+              headerText={t('Delete DataImportCron?')}
+              onDeleteSubmit={() =>
+                k8sDelete({
+                  model: DataImportCronModel,
+                  resource: dataImportCron,
+                  json: undefined,
+                  requestInit: undefined,
```

Comment:
these two attributes were changed to optional in recent sdk release (not sure which one) and can be removed